Code/Editor/2DViewport.cpp

```diff
@@ -234,7 +234,7 @@ void Q2DViewport::UpdateContent(int flags)
 }
 
 //////////////////////////////////////////////////////////////////////////
```

Comment:
I think we can redcode this class btw

Reply:
Haha yes it's on my list 🙂 I'll see if I can get to it next week 👍